frontend/src/components/Nodes/GmailNode/GmailNode.tsx

```diff
@@ -0,0 +1,35 @@
+import { Divider, Typography } from "@mui/material";
+import { CustomNode, CustomNodeProps } from "../CustomNode/CustomNode.tsx";
+
+export const GmailNode = ({
+  nodeTitle = "Gmail",
+  dialogTitle = "Gmail description",
+  dialogContent = (
+    <>
+      A node for integration with Gmail mailbox. It allows you to
```

Comment:
Qua vedi tu cosa mettere ma la frase non scorre benissimo.
```suggestion
      A node connected to the Gmail service. It allows you to
```